test/fixtures/output/connect/offset/describe-offset.golden

```diff
@@ -0,0 +1,7 @@
+Offset Details
++----------+--------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ID       | lcc-123                                                                                                                                                      |
+| Name     | az-connector                                                                                                                                                 |
+| Offsets  | [{"offset":{"event":2,"file":"mysql-bin.000600","pos":2001,"row":1,"server_id":1,"transaction_id":null,"ts_sec":1711788870},"partition":{"server":"dbzv2"}}] |
```

Comment:
If we can't flatten this, let's at the very least prettify this JSON. Take a look at `confluent iam rbac role list` as an example.

Reply:
done